.buildkite/premerge.definition.yaml

```diff
@@ -0,0 +1,11 @@
+agent_queue_id: "v-81e791d9b1a186e5-------1531323149"
+description: spatialos/UnityGDK
```

Comment:
Can't push/pull this via circle yet, because it's hard-coded to our other org, cc @ashleydavies.

Reply:
Will look into shortly - I opened a ticket for this yesterday https://improbableio.atlassian.net/browse/ENG-696

Should be able to get this done before Monday afternoon 👍 I'll prioritise it

Reply:
Was this finished? The ticket's still "In review."